js/app.js

```diff
@@ -81,7 +86,8 @@ const ProjectCardsDetails = [
     id: 'work6',
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/231653944-19089fa7-f62c-4218-a26d-eeb68066bba1.png)

**[OPTIONAL]**

- [ ] It would be nice if you remove all the section descriptions, and unnecessary data from your readme file to make your readme file descriptive and clean :+1: